package.json

```diff
@@ -34,39 +34,39 @@
     "@spark-ui/theme-utils-node": "*"
   },
   "devDependencies": {
-    "@babel/core": "^7.20.12",
-    "@commitlint/cli": "^17.4.2",
-    "@commitlint/config-conventional": "^17.4.2",
-    "@commitlint/config-lerna-scopes": "^17.4.2",
-    "@commitlint/cz-commitlint": "^17.4.2",
-    "@commitlint/prompt-cli": "^17.4.2",
+    "@babel/core": "7.20.12",
+    "@commitlint/cli": "17.4.2",
+    "@commitlint/config-conventional": "17.4.2",
+    "@commitlint/config-lerna-scopes": "17.4.2",
+    "@commitlint/cz-commitlint": "17.4.2",
+    "@commitlint/prompt-cli": "17.4.2",
     "@spark-ui/cli-utils": "*",
-    "@storybook/addon-actions": "^7.0.0-beta.43",
-    "@storybook/addon-essentials": "^7.0.0-beta.43",
-    "@storybook/addon-interactions": "^7.0.0-beta.43",
-    "@storybook/addon-links": "^7.0.0-beta.43",
-    "@storybook/blocks": "^7.0.0-beta.43",
-    "@storybook/builder-vite": "^7.0.0-beta.43",
-    "@storybook/react": "^7.0.0-beta.43",
-    "@storybook/react-vite": "^7.0.0-beta.43",
-    "@storybook/testing-library": "^0.0.14-next.1",
-    "@testing-library/dom": "^8.20.0",
-    "@testing-library/jest-dom": "^5.16.5",
-    "@testing-library/react": "^13.4.0",
-    "@testing-library/user-event": "^14.4.3",
-    "@types/react": "^18.0.27",
-    "@types/react-dom": "^18.0.10",
+    "@storybook/addon-actions": "7.0.0-beta.43",
```

Comment:
all storybook might point to "next" branch, maybe beta, cuz right now we are in x.x.55

Reply:
right!